contracts/stabilizer/PegStabilityModule.sol

```diff
@@ -0,0 +1,305 @@
+pragma solidity ^0.8.4;
+
+import "./../token/Fei.sol";
+import "./../pcv/PCVDeposit.sol";
+import "./../utils/RateLimitedMinter.sol";
+import "./IPegStabilityModule.sol";
+import "./../refs/CoreRef.sol";
+import "./../refs/OracleRef.sol";
+import "../Constants.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+contract PegStabilityModule is IPegStabilityModule, CoreRef, RateLimitedMinter, OracleRef, ReentrancyGuard, PCVDeposit {
+    using Decimal for Decimal.D256;
+    using SafeCast for *;
+    using SafeERC20 for IERC20;
+
+    /// @notice the fee in basis points for selling asset into FEI
+    uint256 public override mintFeeBasisPoints;
+
+    /// @notice the fee in basis points for buying the asset for FEI
+    uint256 public override redeemFeeBasisPoints;
+
+    /// @notice the amount of reserves to be held for redemptions
+    uint256 public override reservesThreshold;
+
+    /// @notice the PCV deposit target
+    IPCVDeposit public override target;
+
+    /// @notice the token this PSM will exchange for FEI
+    /// This token will be set to WETH9 if the bonding curve accepts eth
+    IERC20 public immutable override token;
+
+    /// @notice the max mint and redeem fee in basis points
+    /// Governance can change this fee
+    uint256 public override MAX_FEE = 500;
```

Comment:
Why can governance change the max?

Reply:
I can remove the governor's ability to do this. I added this on the off chance that we want to change the max fee in the future.

Reply:
I think it's fine to leave it in. Fine either way though.